src/messages/FixedTranslationProvider.tsx

```diff
@@ -3,9 +3,8 @@
  *
  * SPDX-License-Identifier: MIT
  */
-import { useSettings } from "../settings/settings";
-import { IntlProvider, MessageFormatElement } from "react-intl";
-import { ReactNode, useEffect, useState } from "react";
+import { ReactNode } from "react";
+import { IntlProvider } from "react-intl";
```

Comment:
Tangentially related lint issue - I used this class in the test harness to trigger the bug. Evidently its normal test use isn't linted. I've made a note to look into that.